app/models/errormetrics/errormetrics.server.ts

```diff
@@ -23,9 +23,7 @@ export const getErrorMetrics = async (publicKeys: string[]): Promise<ErrorMetric
       ",",
     )})&limit=50&order=timestamp.desc&or=(method.neq.synccheck,method.neq.checks)`
 
-  const res = await fetch(metricsURL, {
-    timeout: 1000,
-  })
+  const res = await fetch(metricsURL)
```

Comment:
why did we change this?  Is it necessary?

Reply:
With the updates of Remix, timeout is no longer an argument; TS will fail if you try to send it. @jhodak